features/support/env.rb

```diff
@@ -0,0 +1,20 @@
+require 'fileutils'
+
+def copy_package
+  raise Error.new("bugsnag-source-maps.tgz not found.") unless File.exist?("/app/build/bugsnag-source-maps.tgz")
+  Dir.entries('features/fixtures').reject { |entry| %w[. ..].include?(entry) }.each do |entry|
+    target_dir = "features/fixtures/#{entry}/build"
```

Comment:
This can be simplified a bit:

```suggestion
  Dir['features/fixtures/*'].each do |entry|
    target_dir = "#{entry}/build"
```

```ruby
irb(main):001:0> Dir['features/fixtures/*']
=> ["features/fixtures/rn0_62", "features/fixtures/rn0_63", "features/fixtures/rn0_61", "features/fixtures/rn0_60"]
```